azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -540,32 +603,32 @@ def __init__(self):
     def _reset(self):
         self._request_count = 0  # Total number of vmSettings HTTP requests
         self._error_count = 0   # Total number of errors issuing vmSettings requests (includes all kinds of errors)
-        self._server_error_count = 0  # Count of server side errors (HTTP status in the 500s)
         self._client_error_count = 0  # Count of client side errors (HTTP status in the 400s)
-        self._timeout_count = 0  # Count of timeouts on vmSettings requests
+        self._http_error_count = 0  # Count of HTTP errors other than 400s and 500s
         self._request_failure_count = 0  # Total count of requests that could not be issued (does not include timeouts or requests that were actually issued and failed, for example, with 500 or 400 statuses)
+        self._server_error_count = 0  # Count of server side errors (HTTP status in the 500s)
+        self._timeout_count = 0  # Count of timeouts on vmSettings requests
         self._next_period = datetime.datetime.now() + _VmSettingsErrorReporter._Period
 
     def report_request(self):
         self._request_count += 1
 
-    def report_error(self, error, category=None):
+    def report_error(self, error, category):
         self._error_count += 1
 
-        if self._error_count <= _VmSettingsErrorReporter._MaxLogErrors:
-            logger.info("[VmSettings] [Informational only, the Agent will continue normal operation] {0}", error)
-
-        if self._error_count <= _VmSettingsErrorReporter._MaxTelemetryErrors:
-            add_event(op=WALAEventOperation.VmSettings, message=error, is_success=False, log_event=False)
```

Comment:
Removed these errors from the local log. This was part of 2.7 and was useful for debugging. Now, errors in vmSettings are reported the same way as any other error in the goal state. We are still collecting telemetry while the HGAP stabilizes, but some of this telemetry will also be removed in a future release.